blazor/gantt-chart/performance.md

```diff
@@ -0,0 +1,101 @@
+
+For the most comprehensive performance optimization, combining row, column, and timeline virtualization in the Blazor Gantt Chart allows efficient management of extensive datasets and timelines. By dynamically loading only the visible rows, columns, and timeline cells, this approach minimizes memory usage and ensures smooth scrolling and interaction across large projects. This combined strategy significantly improves performance, reduces initial load times, and enhances the overall user experience by maintaining a responsive and efficient Gantt Chart.
+
+## How to improve performance of Gantt chart in Blazor WASM application
+
+This section provides performance guidelines for using the Syncfusion Gantt Chart component efficiently in Blazor WebAssembly (WASM) applications. For general Blazor WebAssembly performance best practices, please refer to the official guidelines [here](https://learn.microsoft.com/en-us/aspnet/core/blazor/performance?view=aspnetcore-8.0).
+
+1.	[Avoid unnecessary component renders](https://blazor.syncfusion.com/documentation/gantt-chart/webassembly-performance#avoid-unnecessary-component-renders)
+2.	[Avoid unnecessary initial auto-scheduling validation](https://blazor.syncfusion.com/documentation/gantt-chart/webassembly-performance#avoid-unnecessary-initial-auto-scheduling-validation)
+
+## Optimizing performance with AutoCalculateDateScheduling
+
+In the Blazor Gantt Chart component, the start and end dates of tasks are automatically calculated by default based on various factors, including working times, holidays, weekends, and task dependencies (predecessors). While this feature ensures accurate scheduling, it can lead to performance issues when rendering large datasets due to the intensive calculations involved in data validation.
+
+To improve performance when working with large datasets, you can disable this automatic date calculation by setting the [AutoCalculateDateScheduling](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Gantt.SfGantt-1.html#Syncfusion_Blazor_Gantt_SfGantt_1_AutoCalculateDateScheduling) property to `false`. Disabling this feature stops the Gantt Chart from recalculating dates based on the aforementioned factors, thus reducing the processing load and enhancing performance.
+```csharp
+    <SfGantt DataSource="@TaskCollection" AutoCalculateDateScheduling="false">  
+    </SfGantt>
+```
+N> Disabling automatic date calculation by setting the `AutoCalculateDateScheduling` property to `false` can significantly improve the performance of the Blazor Gantt Chart when dealing with large datasets. However, it is crucial to ensure the integrity of your data source, as the Gantt Chart will no longer adjust dates automatically.
```

Comment:
Replace the complete sentence with following one
When setting `Autocalculatedatescheduling` property to false, you must provide the valid data source; otherwise, the Gantt chart will render with invalid dates.